internal/horizon/submit/submitter.go

```diff
@@ -44,6 +45,14 @@ type txFailureResponse struct {
 	} `json:"errors"`
 }
 
+func (f TxFailure) GetLoganFields() map[string]interface{} {
```

Comment:
not necessary but nice to have: since you are casting err (`error` interface) use ptr here